src/screens/createKeys/CreateKeysScreen.tsx

```diff
@@ -33,7 +33,10 @@ export const CreateKeysScreen = ({
           {t('initial_screen_title')}
         </Typography>
       </View>
-      <Typography type={'body3'} style={styles.footerText}>
+      <Typography
+        type={'body3'}
+        style={styles.footerText}
+        color={sharedColors.white}>
```

Comment:
Isn't it supposed to be like `shared.text.primary`?

Reply:
No, this is the onboard screen. This screen should be the same for all diferent themes because we're using an image in background.